.circleci/config.yml

```diff
@@ -236,8 +235,13 @@ commands:
           path: ~/repo
       - restore_cache: *npm_cache_restore_key
       - run:
-          name: npm install
-          command: npm install
+          name: Install dependencies
+          command: |
+            if [[ ! -d node_modules ]]; then
+              npm ci
+            else
+              echo "package.json and package-lock.json unchanged, using cache"
+            fi
```

Comment:
If someone forgot to commit new `package-lock.json` build will be failed https://circleci.com/gh/broadinstitute/ddp-angular/1453